index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="icon" type="image/png" sizes="32x32" href="icon/favicon-32x32.png">
+    <title>Star Wars</title>
+</head>
+<body>
+    <div class="container">
+        <div class="heading">
+            Star Wars
+        </div>   
+        <div class="sub-container">
+            <div class="categories">
+                <span class="category-heading">Categories</span>
+                <span class="category-heading category-heading--btn"><input type="radio" name="categories" >people</span>                 
+                <span class="category-heading category-heading--btn"><input type="radio" name="categories" >planets</span>
+             
+                <span class="category-heading category-heading--btn"><input type="radio" name="categories">Spaceships</span>
+           
+                    <span class="category-heading category-heading--btn"><input type="radio"name="categories">Species</span>
+               
+            </div>
+            <div class="information">
+                <div class="information information__information-label">
+                    <label id="lbl-name">name:</label>
+                    <label id="lbl-Height">Height:</label>
+                    <label id="lbl-Hair-Color">Hair-Color:</label>
+                    <label id="lbl-Skin-Color">Skin-Color:</label>
+                    <label id="lbl-Birth-Year">Birth-Year:</label>
+                    <label id="lbl-Gender">Gender:</label>
+                    <button class="prev-btn">&lt;&lt; previous</button>
+                </div>
+                <div class="information__information-label information__information-label--informationvalue">
```

Comment:
don't concat two words together instead  use - to separate them

Reply:
information__information-label--informationvalue can be written as information__information-label--information-value